docs/annexes/annex-2/annex-2-high-level-requirements.md

```diff
@@ -454,7 +454,7 @@ A.  Requirements regarding attestation formats
 | **Index** | **Requirement specification** |
 |-----------|--------------|
 | ARB_1 | The body responsible for an Attestation Rulebook for a given type of attestation SHALL specify that at least one or more of the following common format(s) must be used for these attestations:                                                            <ul><li>The format specified in ISO/IEC 18013-5, see [ISO18013-5].</li><li>The format specified in "SD-JWT-based Verifiable Credentials (SD-JWT VC)", see [SD-JWT-VC].</li><li>The format specified in "W3C Verifiable Credentials Data Model", see [W3C VCDM v1.1] or [W3C VCDM v2.0]</li><ul> |
-| ARB_2 | The body responsible for an Attestation Rulebook SHALL analyse whether it must be possible for a User to present that type of attestation when the Wallet Instance and the Relying Party are in proximity and attestations are presented without using the internet. If so, the Attestation Rulebook SHALL specify that the attestations must be issued in the ISO/IEC 18013-5-compliant mdoc format. |
+| ARB_2 | The body responsible for an Attestation Rulebook SHALL analyze whether it must be possible for a User to present that type of attestation when the Wallet Instance and the Relying Party are in proximity and attestations are presented without using the internet. If so, the Attestation Rulebook SHALL specify that the attestations must be issued in the ISO/IEC 18013-5-compliant mdoc format. |
```

Comment:
@pinamiranda It looks like this PR changes everything from British to American English. For example we need:

1. `organisation` instead of `organization`
2. `analyse` instead of `analyze` 
3. `recognised` instead of `recognized`

Reply:
Corrected directly in the [branch ](https://github.com/pinamiranda/eudi-doc-architecture-and-reference-framework/tree/consistent-capitalization)